openlibrary/catalog/merge/merge.py

```diff
@@ -30,8 +30,8 @@ def build_amazon(edition, author):
     amazon['isbn'] = editon['isbn_10']
     if 'publish_date' in edition:
         amazon['publish_date'] = merge.amazon_year(edition['publish_date'])
-    if authors:
-        amazon['authors'] = authors
+    if author:
```

Comment:
I recommend leaving this plural and changing the formal in the definition since that's how all the usages and other implementations are. 

Speaking of other implementations, this module is a 90% duplicate with amazon.py, so a better solution might be to sort that out.

Reply:
Opened #1677 so we can deal with the second paragraph in a _separate_ PR.